x/stake/simulation/invariants.go

```diff
@@ -94,17 +92,22 @@ func SupplyInvariants(ck bank.Keeper, k stake.Keeper,
 			},
 		)
 
+		// XXX TODO investigate why go test TestGaiaSimulationAfterImport
+		//          loses decimal fractions if RoundInt is not used below here
```

Comment:
Note this inconsistency

Reply:
(error is in the order of the final decimal places)

Reply:
Might be fixed by changes in https://github.com/cosmos/cosmos-sdk/pull/2982

Reply:
I think we should not round... safer to merge with simulation_after_import failing but the rest passing

Reply:
k I just undid the rounding

Reply:
woot! this seems to have resolved the issue with no more rounding 😎